drltrace/drltrace.c

```diff
@@ -161,7 +161,8 @@ lib_entry(void *wrapcxt, INOUT void **user_data)
             dr_fprintf(outf, "<invalid memory>");
             /* Just keep going */
         });
-        dr_fprintf(outf, ")");
+        dr_fprintf(outf, ") thread id: "TIDFMT" and return to "PFX"",
```

Comment:
How about using the thread id as a prefix, like DrM does, esp if this is going to stderr and then it would be part of the distinguish-from-app-output prefix.  That seems more natural, having it up front.